src/main/java/tech/jhipster/lite/module/infrastructure/secondary/javadependency/gradle/GradleCommandHandler.java

```diff
@@ -260,15 +297,14 @@ private void addPropertyTo(BuildProperty property, Path buildGradleFile) {
 
     fileReplacer.handle(
       projectFolder,
-      ContentReplacers.of(
-        new MandatoryFileReplacer(
-          new JHipsterProjectFilePath(Path.of(projectFolder.folder()).relativize(buildGradleFile).toString()),
-          replacer
-        )
-      )
+      ContentReplacers.of(new MandatoryFileReplacer(new JHipsterProjectFilePath(projectFolderRelativePathFrom(buildGradleFile)), replacer))
     );
   }
 
+  private String projectFolderRelativePathFrom(Path buildGradleFile) {
```

Comment:
How about returning directly a JHipsterProjectFilePath? You seem to always use this method that way
```suggestion
  private JHipsterProjectFilePath projectFolderRelativePathFrom(Path buildGradleFile) {
    return new JHipsterProjectFilePath(Path.of(projectFolder.folder()).relativize(buildGradleFile).toString());
```

Reply:
Sure! Nice 👍